packages/core/src/Camera.ts

```diff
@@ -59,10 +64,36 @@ export class Camera extends Component {
 
   /**
    * Depth texture mode.
+   * If `DepthTextureMode.PrePass` is used, the depth texture can be accessed in the shader using `camera_DepthTexture`.
+   *
    * @defaultValue `DepthTextureMode.None`
    */
   depthTextureMode: DepthTextureMode = DepthTextureMode.None;
 
+  /**
+   * Whether to enable opaque texture.
+   * If enabled, the opaque texture can be accessed in the shader using `camera_OpaqueTexture`.
+   *
+   * @defaultValue `false`
+   *
+   * @remarks If enabled, the `independentCanvasEnabled` property will be forced to be true.
+   */
+  opaqueTextureEnabled: boolean = false;
+
+  /**
+   * Opacity texture down sampling.
+   *
+   * @defaultValue `Downsampling.TwoX`
+   */
+  opaqueTextureDownsampling: Downsampling = Downsampling.TwoX;
+
+  /**
+   * Multi-sample anti-aliasing samples.
+   *
+   * @remarks The `opaqueTextureEnabled` property should be `true` to take effect, otherwise it will be invalid.
+   */
```

Comment:
this comment is error,  this parameter is used to match internal canvas

Reply:
This is a public API

Reply:
done